### PR TITLE
chore: fix and run prettier

### PR DESCRIPTION
**Pull request by sebald**
# Description

Seems like prettier didn't run on some files ... whoops.

@marigold-ui/developer

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -3,27 +3,25 @@
...
 dist
-docs/.cache
-docs/public/render-page.js.map
 LICENSE
 storybook-static
 pnpm-lock.yaml
+public
 
-*.png
+favicon.png
```
Comment (aromko) [inline .prettierignore:24]: Why this change? All png's can be ignored or?

File: config/storybook/.storybook/main.ts [hunk 2]
```diff
@@ -1,9 +1,10 @@
 import type { StorybookConfig } from '@storybook/react-vite';
-import path from 'node:path';
 import { dirname, join } from 'path';
 import { mergeConfig } from 'vite';
 import viteTsConfigPaths from 'vite-tsconfig-paths';
```
Comment (aromko) [inline config/storybook/.storybook/main.ts:5]: I don't like the blank lines between imports. Do we want this?

File: .prettierignore (hunk 1 above)
Comment (sebald) [reply .prettierignore:24]: sadly no :D

https://github.com/marigold-ui/marigold/pull/4055/files#diff-0a384ae6c978e8d201e4f8a3b211a6839a6669586d575a9b14b0065ab099f49b

File: config/storybook/.storybook/main.ts (hunk 2 above)
Comment (sebald) [reply config/storybook/.storybook/main.ts:5]: this is done by the prettier import plugins. you can talk to @sarahgm I don't care :D 

I didn't change the settings of the plugin

Comment (sarahgm) [reply config/storybook/.storybook/main.ts:5]: I don't care

Comment (sebald) [reply config/storybook/.storybook/main.ts:5]: So then change the config in another PR and run the change?

**Review by aromko — APPROVED** [review]
